### DISCUSSION TITLE
Doesn't Projects support how Properties format the frontmatter?

### DISCUSSION THREAD

**Discussion by thoresson** [marcusolsson/obsidian-projects#677] [Help]
I've just started to explore Projects as a way to get better overview of what I am working on. But I'm having trouble with a couple of the properties in my frontmatter. Doesn't Projects support this syntax (yet)?

`
Status:
  - "[[High]]"
`

**Comment by Acylation** [comment]
Quoted syntax is supported. You can set the `Quote string` preference to be `always`. Then, when you type in internal link to the cell using `[[link]]` syntax, the string in note will be quoted, and the linked file will be counted in outgoing link. FYI #186 & #244

<img width="806" alt="image" src="https://github.com/marcusolsson/obsidian-projects/assets/73122375/28d66637-0852-41e0-a885-aa678692abb6">
